libr/core/cmd_print.c

```diff
@@ -5102,21 +5121,22 @@ static int cmd_print(void *data, const char *input) {
 				if ((section = r_bin_get_section_at (obj, core->offset, true))) {
 					vaddr = core->offset + section->vaddr - section->paddr;
 				}
-
-				r_cons_printf ("{\"string\":");
+				pj_o (pj);
 				str = r_str_utf16_encode ((const char *) core->block, len);
-				r_cons_printf ("\"%s\"", str);
-				r_cons_printf (",\"offset\":%"PFMT64u, core->offset);
-				r_cons_printf (",\"section\":\"%s\"", vaddr == UT64_MAX? "unknown": section->name);
-				r_cons_printf (",\"length\":%d", len);
+				pj_krs (pj, "string", str);
```

Comment:
just pass pj_ks and remove the r_str_utf16_encode which is wrong, the only thing you have to do is null terminate that core->block at the given len

Reply:
On making the changes that you said. I am not getting the right output. The new output is as follows:

<img width="679" alt="screen shot 2019-02-24 at 4 57 06 pm" src="https://user-images.githubusercontent.com/20895544/53298535-80746c80-3855-11e9-91e5-48385ac215de.png">